### PR TITLE
Add remix filter

### LINKED ISSUE #17: Add a new filter to exclude remixes
Having the ability to filter out remixes would be a useful way to reduce clutter. 

I think the most obvious and straightforward way to detect a remix release is to check for `remix` string inside a title (case-insensitive). This would be visually similar to the existing `Exclude Various Artists` checkbox filter.

### PR DESCRIPTION
**Pull request by schwartzadev**
Closes #17

### CODE REVIEW

**Comment by jakubito** [comment]
@schwartzadev please rebase this one too

**Comment by schwartzadev** [comment]
done!

File: src/components/filters/RemixesFilter.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { deferred } from 'helpers'
+import { setFilters } from 'state/actions'
+import { getFiltersExcludeRemixes } from 'state/selectors'
+import { Checkbox } from 'components/common'
+
+/**
+ * Render Various Artists filter
```
Comment (jakubito) [inline src/components/filters/RemixesFilter.js:9]: Render remixes filter

File: src/state/selectors.js [hunk 2]
```diff
@@ -255,6 +261,22 @@ const getNonVariousArtistsAlbumIds = createSelector(getAlbumsArray, (albums) =>
   }, /** @type {string[]} */ ([]))
 )
 
+/**
+ * Get all non-"Remix" album IDs
+ */
+const getNonRemixAlbumIds = createSelector(getAlbumsArray, (albums) =>
+  albums.reduce((ids, album) => {
+    console.log(album.name)
```
Comment (jakubito) [inline src/state/selectors.js:269]: Please remove

**Review by jakubito — CHANGES_REQUESTED** [review]

**Comment by jakubito** [comment]
Merged in https://github.com/jakubito/spotify-release-list/commit/552cc4686f2c6dd51396ecad352c7e06426545f8
Thanks

**Pull request closed without merging**